modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -218,44 +218,45 @@ export class Transaction extends BaseTransaction {
   /**
    * Get the list of outputs. Amounts are expressed in absolute value.
    */
-  get stakedOutputs(): Entry[] {
-    // Get staked outputs
-    const addValidatorTx = this.avaxPTransaction as AddValidatorTx;
-    return [
-      {
-        address: addValidatorTx.getNodeIDString(),
-        value: addValidatorTx.getStakeAmount().toString(),
-      },
-    ];
+  get outputs(): Entry[] {
+    if (this.type === TransactionType.Import) {
```

Comment:
might be cleaner with switch statement instead of a list of `if`

Reply:
done